shared/references/cce-redhat-avail.txt

```diff
@@ -1,10 +1,7 @@
-CCE-86034-6
```

Comment:
where is this CCE used?

Reply:
In `mount_option_home_usrquota` for RHEL7